### ISSUE TITLE
Place clock on the far-right

### ISSUE THREAD

**Issue by luispabon** [hardpixel/unite-shell#90]
As with Unity. ATM clock position changes depending on whatever else is on the tray, which negates eye-muscle-memory.

**Comment by Natetronn** [comment]
Or allow us to put it back in the center, like it was in 18.04

**Comment by jonian** [comment]
@Natetronn The extension provides the setting "Extend left box". If you disable it, the clock will be reset to the center.

**Comment by Valeryan24** [comment]
Yes, your fantastic extension (that I procrastinated very much too long to test and keep !) helped me to come back to some of the best Unity features I missed, and get rid of lots other addons (some of them not maintained anymore and not compatible with Gnome Shell 3.36) for just Unite.

But there is one I still need, Frippery Move Clock. So please it would be very great to have an option to put clock and date on the top right, just before the aggregate menu (which would still be at the right side), but on the right of all other icons / indicators, or (for people preferring this) on the very right, also after aggregate menu.

Then it would be perfect !

**Comment by ingria** [comment]
> The extension provides the setting "Extend left box". If you disable it, the clock will be reset to the center

That's not exactly correct behavior. If you simply extend left box, clock position will depend on tray icon count. Clock will "jump" left and right when new tray icons arrive/disappear.

Clock should be on its fixed place all the time - like on all other OSes and old Unity.

So, I suggest placing clock on the left of the aggregate menu, like in old Unity.
